civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -64,6 +65,7 @@
             appendTo: $rootElement,
             templateUrl: config.path.TPL + 'modal/document.html?v=3',
             controller: 'ModalDocumentCtrl',
+            controllerAs: 'documentModal',
```

Comment:
It was affected, when I implemented controllerAs syntax in T&A, and the document modal view was broken in SSP. So it actually fixes the problem rather than creating one.